fundamentals-recap/practice-fundamentals-vocabulary.md

```diff
@@ -0,0 +1,190 @@
+# Practice: Fundamentals Vocabulary
+
+## Directions
+
+Complete these questions. Retry questions until you have the correct answer on each. Use any resource to answer these questions (including notes, other code, the Internet, assistance from others).
+
+When you finish, write down any remaining questions you have in your notes, and bring them to class.
+
+# Practice!
+
+<!-- Question 1 -->
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: 82df3237-9f64-4ddc-b5a2-843f53430b15
+* title: Fundamentals Vocabulary Review I
+
+##### !question
+
+What is a statement?
+
+##### !end-question
+
+##### !options
+* a unit of code that can be evaluated into one value
+* a unit of code that performs an action
+* a value
+
+##### !end-options
+
+##### !answer
+* a unit of code that performs an action
+##### !end-answer
+
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: 3a91ff89-ad35-4f18-bb44-fccd104677dc
+* title: Fundamentals Vocabulary Review II
+
+##### !question
+
+What is the expression in this code snippet?
+
+```python
+if score > 150000:
+   print(“New High Score!”)
+```
+##### !end-question
+
+##### !options
+
+* `if score > 150000:`
+* `print("New High Score!")`
+* `score > 150000`
+
+##### !end-options
+
+##### !answer
+
+* `score > 150000`
+
+##### !end-answer
+
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: 5a5b2aa8-08ef-4a98-bf3c-e1a61461ceaa
+* title: Data Types review
+
+##### !question
+
+The code snippet below returns a `TypeError: can't multiply sequence by non-int of type 'str’` which means we cannot multiply strings. Let's check the data type of the inputs, starting with `n1`. Which option holds the syntax we use to check `n1`'s data type in a print statement?
+
+##### !end-question
+
+##### !options
+
+* `type = n1`
+* `type(n1)`
+* `type.n1()`
+* `n1 = type()`
+
+##### !end-options
+
+##### !answer
+
+* `type(n1)`
+
+##### !end-answer
+
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: checkbox
+* id: 8174678f-cf23-4605-a033-91c957a2cf81
+* title: Casting Review
+
+##### !question
+
```

Comment:
@audreyandoy What was the intended question you had here? ? The text is missing!